sql/com.delighted/survey_response_created_1.sql

```diff
@@ -0,0 +1,50 @@
+-- AUTO-GENERATED BY igluctl DO NOT EDIT
+-- Generator: igluctl 0.2.0
+-- Generated: 2017-10-30 10:40
+
+CREATE SCHEMA IF NOT EXISTS atomic;
+
+CREATE TABLE IF NOT EXISTS atomic.com_delighted_survey_response_created_1 (
+    "schema_vendor"                                 VARCHAR(128)  ENCODE ZSTD NOT NULL,
+    "schema_name"                                   VARCHAR(128)  ENCODE ZSTD NOT NULL,
+    "schema_format"                                 VARCHAR(128)  ENCODE ZSTD NOT NULL,
+    "schema_version"                                VARCHAR(128)  ENCODE ZSTD NOT NULL,
+    "root_id"                                       CHAR(36)      ENCODE RAW  NOT NULL,
+    "root_tstamp"                                   TIMESTAMP     ENCODE ZSTD NOT NULL,
+    "ref_root"                                      VARCHAR(255)  ENCODE ZSTD NOT NULL,
+    "ref_tree"                                      VARCHAR(1500) ENCODE ZSTD NOT NULL,
+    "ref_parent"                                    VARCHAR(255)  ENCODE ZSTD NOT NULL,
+    "event_data.comment"                            VARCHAR(512)  ENCODE ZSTD,
```

Comment:
I suspect a lot of these fields should be `NOT NULL` - to trigger this in Igluctl, you will need to add those fields into the `required: []` array.